x-pack/plugin/sql/jdbc/qa/security/build.gradle

```diff
@@ -0,0 +1,63 @@
+dependencies {
+    testCompile project(':x-pack:plugin:core')
+}
+
+Project mainProject = project
+
+group = "${group}.x-pack.qa.sql.security"
+
+configurations.create('testArtifacts')
+
+TaskProvider testJar = tasks.register("testJar", Jar) {
+    appendix 'test'
+    from sourceSets.test.output
+}
+
+artifacts {
+    testArtifacts testJar
+}
+
+// Tests are pushed down to subprojects and will be checked there.
+testingConventions.enabled = false
+
+subprojects {
```

Comment:
Let's please not do this anymore. Instead of creating subprojects here, let's just define two separate test clusters. This mess of sharing test artifacts just to run tests two different ways is completely unnecessary. We should refactor the SQL project to ditch this but at least for this PR, let's not copy a bad pattern. Let me know if you need help refactoring this.

Reply:
As discussed we'd rather keep it like that to have the parallel execution of the single-node/mutli-node etc. cases